src/SmartComponents/SystemAdvisor/SystemAdvisor.js

```diff
@@ -176,16 +179,28 @@ const BaseSystemAdvisor = () => {
     kbaLoading = false,
     isFirstLoad = false
   ) => {
-    const builtRows = activeReports.flatMap((value, key) => {
+    let isRecParamPresent =
+      window.location.href.indexOf('recommendations') > -1 ? true : false;
+    let newActiveReportsList = activeReports;
+
+    if (isRecParamPresent) {
+      let activeRec = location[2];
```

Comment:
Does this work for both beta/non-beta?

Reply:
Yeah, this is very similar to how breadcrumbs currently work